include/hybridmap/NeutralAtomDefinitions.hpp

```diff
@@ -0,0 +1,40 @@
+//
+// This file is part of the MQT QMAP library released under the MIT license.
+// See README.md or go to https://github.com/cda-tum/qmap for more information.
+//
+
+#pragma once
+
+#include "Definitions.hpp"
+
+#include <cstdint>
+#include <set>
+#include <utility>
+#include <vector>
+
+namespace qc {
+// A CoordIndex corresponds to node in the SLM grid, where an atom can be placed
+// (or not).
+using CoordIndex   = std::uint32_t;
+using CoordIndices = std::vector<CoordIndex>;
+// A HwQubit corresponds to an atom in the neutral atom architecture. It can be
+// used as qubit or not and occupies a certain position in the architecture.
+using HwQubit                      = uint32_t;
+using HwQubits                     = std::set<HwQubit>;
+using HwPositions [[maybe_unused]] = std::vector<HwQubits>;
+// A Qubit corresponds to a qubit in the quantum circuit. It can be mapped to a
+// hardware qubit.
+using Qubits = std::set<Qubit>;
```

Comment:
The reason for that is that op->getUsedQubit returns a set and I didn't bother converting. Is this still worth it even if I always have to convert?

Reply:
Ah, I see. In that case, not immediately.
It might be worth to add another utility function to the `Operation` class though that returns an `std::unordered_set`.

Reply:
Ok, then I would leave it for the moment, if that's fine

Reply:
Sure, feel free to resolve here. I doubt that this really is a performance bottleneck in the current mapper. It might just be worth keeping in mind.

Reply:
I'll remember it for the future. When I write the code I barely knew anything about C++, so there a lot of things I would do differently next time 👍